src/main/webapp/app/core/course/manage/control-center/control-center.component.html

```diff
@@ -0,0 +1,13 @@
+@if (course().isAtLeastInstructor && this.irisEnabled()) {
+    <div class="card mb-3" style="max-width: 35rem; width: 100%; min-height: 91%">
```

Comment:
Should we define these styles in the scss file rather than inline?
(or, alternatively, use pre-defined classes)